src/main/java/com/example/airbnbapi/handler/HandleInput.java

```diff
@@ -14,32 +14,51 @@
     @Override
     public void run(String... args) throws Exception {
 
-        System.out.println("What is the name of your JSON file?: ");
-        String resourceFilePath = input.next();
-        System.out.println("--------");
+        boolean quit = false;
+        FetchObject fetch = new FetchObject();
```

Comment:
Consider `@Autowire`-ing this object into the class, rather than using the explicit `new` key word